lib/pages/chat/events/message/message_content_with_timestamp_builder.dart

```diff
@@ -92,8 +88,7 @@ class MessageContentWithTimestampBuilder extends StatelessWidget {
             ),
             child: MultiPlatformSelectionMode(
               event: event,
-              longPressSelect: longPressSelect,
-              useInkWell: !PlatformInfos.isWeb,
+              isClickable: responsiveUtils.isMobileOrTablet(context),
```

Comment:
two condition is completely different, please be careful. This will disable selection in tablet and mobile web

Reply:
But as i tested again, it works fine.

Reply:
I want it to be clickable when it's not a big screen basically so I don't bother if it's on mobile or web I'm just checking screen dimensions. IMO it's fine as it is